examples/next/pages/ui/components/ai/ai-conversation/amplify/data/resource.ts

```diff
@@ -0,0 +1,34 @@
+import { a, ClientSchema, defineData } from '@aws-amplify/backend';
+
+const schema = a.schema({
+  Foo: a
+    .model({ bar: a.integer() })
+    .authorization((allow: { authenticated: () => any }) =>
+      allow.authenticated()
+    ),
+
+  pirateChat: a.conversation({
+    // better dx for aiModel in progress
```

Comment:
should this be a //@TODO so it's searchable?

Reply:
This was just copied from Ian's testing setup and so is not finalized yet, eventually will go back and update all these backend definitions, will add a todo for that